src/main/resources/Sessions/Sessions.drl

```diff
@@ -24,7 +24,7 @@ rule "Session Time For the Judge not Available"
     when
         $s : Session(judgeId != null)
         $j : Judge(id == $s.judgeId)
-        not Availability(judgeId == $s.judgeId && ($s.start.compareTo(start) >= 0 && $s.end.compareTo(end) <= 0))
+        not Availability(judgeId == $s.judgeId && ($s.start.compareTo(start) >= 0 && $s.getEnd().compareTo(getEnd()) <= 0))
```

Comment:
that is ok. rules is not java (it is java like style) internally it uses getter anyway, so this change does not really change anything